docs/overview/binaries.rst

```diff
@@ -24,23 +24,23 @@ unpacked from cache. Such cache can be used locally or uploaded to server.
 Details
 =======
 
-Default server with binaries is
+The default server with cached binaries is
 `ingenue/hunter-cache <https://github.com/ingenue/hunter-cache>`__.
-Archives saved as assets to `cache tag <https://github.com/ingenue/hunter-cache/releases/tag/cache>`__.
-Uploaded packages can be found by ``upload.*`` query on page with branches:
+Archives are tagged as assets via a git `tag <https://github.com/ingenue/hunter-cache/tags>`__.
```

Comment:
I took another stab at it.  This is an area I'm not very familiar with.  If it's not headed the right direction, it may be better for me to back away from this part for now.